### PR TITLE
Implemented cmap.

### PR DESCRIPTION
**Pull request by pcubillos**
Hi Francesco,
I was looking for a piece of code that transformed an RGBA colormap to an RGB colormap.  I couldn't find a solution, but I found your repo, and combined it with another piece of code to implement this new function.

### CODE REVIEW

**Comment by montefra** [comment]
@pcubillos: thank you very much for the PR. It's very pleasant to know that someone uses this little project. I'll accept it for sure after checking the code.

File: mimic_alpha.py [hunk 1]
```diff
@@ -189,6 +191,81 @@ def colorAlpha_to_rgb(colors, alpha, bg='w'):
...
+    Parameters:
+    -----------
+    cmap_name: String
+       A standard Matplotlib colormap name:
+       http://matplotlib.org/examples/color/colormaps_reference.html
+    alpha: Float
+       Value of alpha to mimic in range [0,1].
+    bg: Matplotlib color
+       Color of the background.
+
```
Comment (montefra) [inline mimic_alpha.py]: Add the return value, please:

```
Returns
----------
ma_cmap : :class:`matplotlib.colors.Colormap`
     add your explanation
```

File: mimic_alpha.py [hunk 2]
```diff
@@ -189,6 +191,81 @@ def colorAlpha_to_rgb(colors, alpha, bg='w'):
...
+    # Convert RGBA colors from cmap into RGB:
+    cdict = {'red':[], 'green':[], 'blue':[]}
+    for pos in position:
+        r, g, b = colorAlpha_to_rgb(input_cmap(pos), alpha, bg)[0]
+        cdict['red'  ].append((pos, r, r))
+        cdict['green'].append((pos, g, g))
+        cdict['blue' ].append((pos, b, b))
+
+    # mimic-alpha colormap:
+    ma_cmap = mplc.LinearSegmentedColormap('my_colormap', cdict, 256)
```
Comment (montefra) [inline mimic_alpha.py]: I would make the the name of the color map an optional argument of the function.
Something like: 

```
def cmap(cmap_name, alpha, bg="w", out_cmap_name=None):
[...]
if not out_cmap_name:
    out_cmap_name = cmap_name + "_{0:.1f}".format(alpha)
ma_cmap = mplc.LinearSegmentedColormap(out_cmap_name, cdict, 256)
[...]
```

**Comment by pcubillos** [comment]
Yes, I wasn't careful about that line with the output colormap name. I like your solution.
I'll update the changes and make a new pull request.
Thanks!

**Comment by pcubillos** [comment]
OK, I implemented the changes. I also implemented (and tested) the set_under, set_over, and set_bad optional arguments, and bumped the version up to 0.22

cheers,
Patricio.

**Comment by montefra** [comment]
Thanks you very much for the addition. I'll merge a soon as I'm in front of a computer.

**Pull request merged**